setup.py

```diff
@@ -121,6 +121,9 @@ def get_description_type(path=PKG_DESCRIBE):
     "download_url": "{}/tarball/v{}".format(REPOSITORY, get_version()),
     "packages": find_packages(where=PROJECT, exclude=EXCLUDES),
     "install_requires": list(get_requires()),
+    "extras_require": {
+        "ml": ["pandas==2.1.2"],
```

Comment:
The idea is that we can make this an extra package e.g. `pip install pyensign[ml]`, I'm not sure what else we need to do to make that happen but this ensures that pandas is not in the regular dependencies.